### ISSUE TITLE
[Bug]: While an app is running if i switch off the accessability config the whole system becomest hang.

### ISSUE THREAD

**Issue by vitalii-zinchenko** [vitalii-zinchenko/dictara#50]
When a user have running app on the Mac then the user goes and disables the accessibility for the Dictar and at that time everything freezes. Like I can't mouse, cannot click to anything, everything freezes.
